dots/vimrc

```diff
@@ -161,43 +166,59 @@ let g:coffee_compile_vert=1
 nnoremap j gj
 nnoremap k gk
 
-" Visually select the text that was last edited/pasted
-nnoremap gV `[v`]
-
 " Move between splits
 noremap <C-h> <C-w>h
 noremap <C-j> <C-w>j
 noremap <C-k> <C-w>k
 noremap <C-l> <C-w>l
 
+" NERDTree in a buffer (like netrw)
+nnoremap <silent>- :silent edit %:p:h<CR>
+nnoremap <silent>_ :silent edit .<CR>
+
+" Override jumplist commands
+nnoremap <silent><C-i> :BufSurfBack<CR>
+nnoremap <silent><C-o> :BufSurfForward<CR>
+
```

Comment:
This overrides existing behavior. These jumplist never make sense to me until we started using the [ton/vim-bufsurf](https://github.com/ton/vim-bufsurf) plugin. Just testing these to see if it makes sense to override them.